Solutions/Digital Shadows/Data Connectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -18,22 +18,30 @@ def __init__(self, id, key, secret, url):
         self.id = id
```

Comment:
I have renamed the folder, but I think now if we create a PR it will be like a new Solution for the reviewers. This means they will check all of the things again and it would take lot of time like the current PR which is open for main Azure Sentinel repo.